LEAF_Request_Portal/sources/Service.php

```diff
@@ -173,8 +173,8 @@ public function addMember($groupID, $member)
             {
                 $sql_vars = array(':userID' => $member,
                               ':groupID' => $groupID, );
-                $this->db->prepared_query('INSERT INTO users (userID, groupID)
-                                                VALUES (:userID, :groupID)', $sql_vars);
+                $this->db->prepared_query('INSERT INTO users (userID, groupID, backupID)
```

Comment:
This goes back to making the modification on the database itself.  Otherwise you have to make sure every single insert has the empty value.

Reply:
I did make sure of that but added the default value anyway, that is a better way of doing it.